File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantBatchesCreatePlantBatchesWasteRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantBatchesCreatePlantBatchesWasteRequest {
    @JsonProperty("MixedMaterial")
    public String mixedMaterial;
    @JsonProperty("Note")
    public String note;
    @JsonProperty("PlantBatchName")
    public String plantBatchName;
    @JsonProperty("ReasonName")
    public String reasonName;
    @JsonProperty("UnitOfMeasureName")
    public String unitOfMeasureName;
    @JsonProperty("WasteDate")
    public String wasteDate;
    @JsonProperty("WasteMethodName")
    public String wasteMethodName;
    @JsonProperty("WasteWeight")
    public Double wasteWeight;
}

File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/PlantBatchesCreatePlantBatchesWasteRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class PlantBatchesCreatePlantBatchesWasteRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("MixedMaterial")]
        public string? MixedMaterial { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Note")]
        public string? Note { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("PlantBatchName")]
        public string? PlantBatchName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("ReasonName")]
        public string? ReasonName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("UnitOfMeasureName")]
        public string? UnitOfMeasureName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("WasteDate")]
        public string? WasteDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("WasteMethodName")]
        public string? WasteMethodName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("WasteWeight")]
        public double? WasteWeight { get; set; }
    }
}
